src/LinkBubbleMenu/EditLinkMenuContent.tsx

```diff
@@ -109,6 +109,12 @@ export default function EditLinkMenuContent({
     // current site). It also allows the value to pass browser-builtin
     // `type="url"` validation.
     let currentHrefValue = hrefRef.current.value.trim();
+
+    // If the user types in a relative path, don't format it.
+    if (currentHrefValue.startsWith("/")) {
+      return;
```

Comment:
Hm, I don't think we should do an early return here. I think it is still useful to `trim()` the value and use `encodeurl` (so we can handle invalid URL characters). I think we could just change the conditional below that adds `http://` to instead check:

```ts
    if (
      currentHrefValue &&
      !currentHrefValue.startsWith("/") &&
      !currentHrefValue.startsWith("http://") &&
      ...
```